### DISCUSSION TITLE
How to stretch/not keep scale a image

### DISCUSSION THREAD

**Discussion by Matiz7** [ImageMagick/ImageMagick6#338] [General] [answered]
### ImageMagick version

6.9.10-86

### Operating system, version and so on

centos7

### Description

when I use resize param It's always keep scale
if I dont want scale   how to do?

**Comment by snibgo** [comment]
I don't understand the question. Please show the command you used, sample input and output files, and say why the result is not what you want.

**Comment by fmw42** [comment, marked as answer]
add ! to the argument in your resize. (-resize WxH!) to stretch and distort.
